pkg/operator/ceph/csi/csidriver.go

```diff
@@ -95,7 +97,7 @@ func (d v1CsiDriver) reCreateCSIDriverInfo(ctx context.Context) error {
 		return errors.Wrapf(err, "failed to delete CSIDriver object for driver %q", csiDriver.Name)
 	}
 	logger.Infof("CSIDriver object deleted for driver %q", csiDriver.Name)
-	_, err = csiClient.Create(ctx, d.csiDriver, metav1.CreateOptions{})
```

Comment:
How about leaving this `d.csiDriver` reference and delete the two local vars `csiDriver` and `csiClient`? The local vars are not necessary IMO.